src/main/graphql/GetConsignment.graphql

```diff
@@ -0,0 +1,7 @@
+ query getConsignment($consignmentid: Long) {
+   getConsignment(consignmentid: $consignmentId) {
+     userid
+       seriesid
```

Comment:
Adjust the alignment of "seriesid"

Reply:
> Looks OK. One very minor issue. I assume once this is in place you will make the necessary changes to the TDR frontend to add the checks and error handling?

I'll make that change and push again asap. Yeah wanted to make sure to just focus on one thing at a time since the work will be spread across different repos.